Avatar/avatar.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./avatar.css" />
+</head>
+
+<body>
+    <nav class="nav-header">
+        <img src="../Images/component-logo.svg" alt="vodka-ui" />
+        <div class="nav-items">
+            <a href="#">Installation</a>
+            <a class="active" href="#">Docs</a>
+            <a href="#">Releases</a>
+        </div>
+    </nav>
+    <section class="left-right-combine">
+        <aside class="index-of-component">
+            <h1 class="index-heading">Components</h1>
+            <ul>
+                <li><a href="#">Introduction</a></li>
+                <li><a href="#" class="active">Avatar</a></li>
+                <li><a href="#">Alert</a></li>
+                <li><a href="#">Badge</a></li>
+                <li><a href="#">Button</a></li>
+                <li><a href="#">Card</a></li>
+                <li><a href="#">Image</a></li>
+                <li><a href="#">Input</a></li>
+                <li><a href="#">List</a></li>
+                <li><a href="#">Toast</a></li>
+            </ul>
+        </aside>
+        <section class="right-side-of-page">
+            <main class="content-main">
+                <h1 class="content-heading">Avatar</h1>
+                <p class="content-paragraph">
+                    Avatars are images that users can set as their profile picture. On GitHub, they're always going to be rounded squares. They can be custom photos, uploaded by users, or generated as Identicons as a placeholder.
+                </p>
+                <h1 class="content-heading">Profile Avatar</h1>
+                <p class="content-paragraph">
+                    Profile avatars can be created by adding class
+                    <span class="highlight-word">avatar</span> in img tag.
+                </p>
+                <div class="example-div">
+                    <img class="avatar" src="https://images.unsplash.com/photo-1593085512500-5d55148d6f0d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&au" alt="profile avatar" />
```

Comment:
Would suggest using images after adding them to your assets or images/ folder in your project.